### ISSUE TITLE
Support for INDIGO Sequencer.js

### ISSUE THREAD

**Issue by Paolo97Gll** [indigo-astronomy/indigo_imager#15]
Hi, I have a question. Do you have any plan of integrating the INDIGO Sequencer.js features (described [here](https://github.com/indigo-astronomy/indigo/blob/master/indigo_drivers/agent_scripting/library/Sequencer.md)) in Ain INDIGO Imager like [in the example shown at the end of the documentation](https://github.com/indigo-astronomy/indigo/blob/master/indigo_drivers/agent_scripting/library/Sequencer.md#examples-of-visual-representation-of-a-sequence-in-gui-pre-release-of-indigo-a1v5)? I know there is already a sequence editor, but is much less powerful than the Sequencer.js. For example, now in AIN there is no support for automatic meridian flip.

**Comment by rumengb** [comment]
yes. I am working on this. the thing is that I have so many rhings to do in
the indigo framework that I can not find enough time to add this to Ain. I
need to completely change the sequencer in Ain for that matter. but it is
coming.

**Comment by Paolo97Gll** [comment]
I understand, thank you! If you need help in implementing or testing these
features let me know!

--
Paolo Galli


Il giorno gio 7 mar 2024 alle 15:46 Rumen Bogdanovski <
***@***.***> ha scritto:

> yes. I am working on this. the thing is that I have so many rhings to do
> in
> the indigo framework that I can not find enough time to add this to Ain. I
> need to completely change the sequencer in Ain for that matter. but it is
> coming.
>

**Comment by rumengb** [comment]
Almost ready! :)
![Image](https://github.com/user-attachments/assets/47466da8-08d5-4cdb-8e55-20a253815c27)

**Comment by Paolo97Gll** [comment]
Thanks for the integration! I guess this issue can be closed. We are testing this functionality, so if we find any problems, we will tell you.
